README.md

```diff
@@ -17,48 +17,40 @@ Your One-Stop solution for a full-stack app with ES6/ES2015 React.js featuring u
+- [**Redux**](https://github.com/reactjs/redux) Predictive state containers.
+- Server-side rendering with [**React Router**](https://github.com/reactjs/react-router) 2.x. Having server-side rendering allows you to pre-render the initial state of your components when a user (or search engine crawler) requests a page.
+- Integrating Redux with React Router with ~~Redux Simple Router~~ [React Router Redux](https://github.com/reactjs/react-router-redux)
+- Asynchronous Data Fetching on server-side rendering
+- Server side authentication + Redirecting for components
 - Hot reloading using [**react-transform-hmr**](https://github.com/gaearon/react-transform-hmr)
 - Time travel using [**Redux-Devtools Chrome Extension**](https://github.com/zalmoxisus/redux-devtools-extension)
 - [**Webpack**](https://github.com/webpack/webpack) for both development and production bundles. It's (in my opinion) the best bundler for JS, CSS, LESS, images, and lots more!
 - [**CSS Modules**](https://github.com/css-modules/css-modules) allows for modular and reusable CSS. Say goodbye to conflicts (most of them) and global scope
+
 - **Unit Tests** with webpack, karma, jsdom, mocha, & sinon
 	- Reducers
 	- Components
 	- Synchronous and Asynchronous Actions
+
 - Express 4.x server with a ton of middleware
 - Mongoose for MongoDB
 - Sequelize for Postgres
 - Procfile to enable deployment to Heroku & Docs on Salt configurations + Deployment for Digital Ocean
 
-**Easter Eggs** :egg:
-
-This boilerplate has gone through an evolution
-```
-React.js -> Facebook Flux -> Alt -> Redux
-```
-
-We have two implementations of universal flux:
-- **Redux** is on our active **master** branch
-- [**Alt**](https://github.com/goatslacker/alt) (previously implemented) on [flux/alt](https://github.com/choonkending/react-webpack-node/tree/flux/alt) branch. It features [iso](https://github.com/goatslacker/iso), react-router and ImmutableJS.
-
-> Note: If you have previously used an alt implementation of this repository, please refer to this branch. I will not be updating it as frequently as master, but definitely welcome suggestions!
 
 ## Motivation
 
-The aim of this repo is to incorporate the best practices to building a non-trivial, performant, secure and quality full-stack apps with React.js and Webpack and Node (hence...react-webpack-node). However, along the way we definitely have had extensive additions to this boilerplate! I am working to document this repo extensively so it would be easy for both beginners and experts to begin dev-ing on it without pulling your hair out.
+The motivation is simple, best practices. Everyone's ultimate goal is to build a non-trivial applications. Performant and free of bugs, we want our applications to be secure and joy for the user. We believe that using a mixture of with React.js and Webpack and Node is the best way to accomplish this. (hence...react-webpack-node).
```

Comment:
I read it a couple of times, but I still like the first sentence (hehe). That being said, I prefer your elaboration than the original.

> The aim of this repo is to incorporate the best practices to building a non-trivial, performant, secure and quality full-stack apps with React.js and Webpack and Node

Perhaps we could change it to:

The motivation is simple - best practices. 

We aim to incorporate **best practices** to building a **non-trivial**, performant, secure and bug-free full-stack application. User and developer experience should be delightful!  We believe that using a mixture of React.js, Webpack and Node is the best way to accomplish this. (hence...react-webpack-node). ✌️

This boilerplate serves a second purpose as well. It is a learning tool for any and all that are interested in learning how to put a large React application together. We have had extensive additions to this boilerplate from the community over time as practices have evolved, and we are always interested in hearing new ideas and welcome all forms of help!